pkg/integration/integration.go

```diff
@@ -108,11 +112,15 @@ func (*Integration) Deactivate(name string, namespace string) error {
 	// This might be a bad idea, but we cannot reference analyzer here
 	foundFilter := false
 	for i, v := range activeFilters {
-		if v == integrations[name].GetAnalyzerName() {
-			foundFilter = true
-			activeFilters = append(activeFilters[:i], activeFilters[i+1:]...)
-			break
+
+		for _, intanal := range integrations[name].GetAnalyzerName() {
```

Comment:
Should we consider renaming this variable to `integrationAnalyzer` ? It might be a bit longer, but I believe it would be more comprehensible.